exporter/clients.go

```diff
@@ -48,6 +61,16 @@ func parseClientListString(clientInfo string) ([]string, bool) {
 
 }
 
+func durationFieldToTimestamp(field string) (int64, error) {
+	parsed, err := strconv.ParseInt(field, 10, 64)
```

Comment:
strconv.Atoi() is the more commonly used function, can you use that?

Reply:
as described before, FormatInt and ParseInt are better fits for this purpose.